test/helpers/AsymmetricKey.ts

```diff
@@ -0,0 +1,65 @@
+export const AsymmetricPrivateKey = `-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I know people don't like keys in repos but this is having actual keys available is the only way to fully test the implementation and having them here is the way to keep tests simple. The two keys available here were generated exclusively for the tests here.

Reply:
Great suggestion. I decided to spend some more time to keep this "clean" of hard-coded keys. I will update the PR soon with a key-generating function using crypto.